packages/transaction-messages/src/compile/message.ts

```diff
@@ -91,5 +91,20 @@ export function compileTransactionMessage(
         lifetimeToken: getCompiledLifetimeToken(lifetimeConstraint),
```

Comment:
Isn't this straight up optional now? Meaning this and the part above can be completely replaced with:


```suggestion
        ...((transactionMessage as Partial<TransactionMessageWithLifetime>).lifetimeConstraint
            ? { lifetimeToken: getCompiledLifetimeToken(transactionMessage.lifetimeConstraint) }
            : null),
            
```